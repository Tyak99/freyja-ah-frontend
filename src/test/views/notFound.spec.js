import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Notfound from '../../views/NotFound';

const setUp = (props = {}) => {
  const component = shallow(<Notfound {...props} />);
  return component;
};
describe('Notfound component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
  it('should render children', () => {
    const wrapper = component.find('h2');
    expect(wrapper.exists()).toBe(true);
  });
});
