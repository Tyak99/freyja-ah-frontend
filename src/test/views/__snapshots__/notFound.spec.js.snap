// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notfound component should render without errors 1`] = `
<div>
  <h2>
     Page not found 
  </h2>
</div>
`;
